rpgJavaInterpreter-core/src/test/resources/QILEGEN/£JAX_D0.rpgle

```diff
@@ -56,9 +56,9 @@
      D £JaxLLb                        5  0 INZ
      D £JaxLLc                        5  0 INZ
      D £JaxLBu                      140    INZ
-     D £JaxLSi                         Z   INZ
-     D £JaxLSt                         Z   INZ
-     D £JaxLEt                         Z   INZ
+     D***> £JaxLSi                         Z   INZ
```

Comment:
Why?

Reply:
Due to conversation with Marco Benetti, we decided to temporarly remove fields defined as 'Z' type (timestamp).